python/lsst/ts/mtdome/mock_llc/amcs.py

```diff
@@ -350,10 +588,19 @@ async def exit_fault(self, start_tai: float) -> float:
         `float`
             The expected duration of the command [s].
         """
-        duration = self.azimuth_motion.exit_fault(start_tai)
-        self.end_tai = start_tai + duration
+        if True in self.drives_in_error_state:
```

Comment:
```
if self.drives_in_error_state:
```

Reply:
That will not work. `self.drives_in_error_state` is an array so the code needs to test if at least one of the elements of the array is True.

Reply:
You can try `any()`. If it is a numpy array, there is an `any()` function as well.